netlify-plugin-cloudinary/src/lib/cloudinary.ts

```diff
@@ -206,20 +207,31 @@ export async function getCloudinaryUrl(options: CloudinaryOptions) {
     if (canSignUpload) {
       // We need an API Key and Secret to use signed uploading
 
-      results = await cloudinary.uploader.upload(fullPath, {
-        ...uploadOptions,
-      })
+      try {
+        results = await cloudinary.uploader.upload(fullPath, {
+          ...uploadOptions,
+        })
+      } catch (error) {
+        console.error(`[Cloudinary] ${ERROR_ASSET_UPLOAD}`)
+        console.error(`[Cloudinary] \tpath: ${fullPath}`)
```

Comment:
is the `\t` mistakenly here? only reason i ask is because it's not in the `unsigned_upload` one. im not sure what that does

Reply:
I added, and missed the other case. Is just for formatting the error. With that `tab` the console.log displays this

```
[Cloudinary] Error uploading assets
    [Cloudinary] path: the_full_path
```